studio/pages/api/props/project/[ref]/api.ts

```diff
@@ -72,7 +72,7 @@ const handleGetAll = async (req: NextApiRequest, res: NextApiResponse) => {
         realtime_enabled: true,
       },
       endpoint: process.env.SUPABASE_URL,
-      restUrl: process.env.SUPABASE_REST_URL,
+      restUrl: `${process.env.SUPABASE_REST_URL}/rest/v1/`,
```

Comment:
Can we revert this change? The original problem with misconfigured storage route has already been addressed. So this might not be relevant anymore. 

In addition, changing restUrl also requires updating the cli to change its env var when updating to newer studio image. Technically a braking change.